packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -153,7 +154,7 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
   return (
     <div ref={containerRef} className={mergeStyles(containerDivStyles)}>
       <Stack verticalFill horizontalAlign="stretch" className={containerClassName} data-ui-id={props.dataUiId}>
-        <Stack horizontal grow>
+        <Stack horizontal grow styles={callArrangementContainerStyles}>
```

Comment:
I'm surprised the control bar still stacks vertically in this Stack because `horizontal` prop is true. Without the  flexDirection: 'column-reverse' style would the control bar still stack horizontally?
Does moving the CallControl in this Stack avoid initial focus before the first tab?

Reply:
That's right, without the column-reverse the control bar would stack horizontally.
![image](https://user-images.githubusercontent.com/97130533/207403310-22eee1e8-be9f-4b80-b377-1ccfcaf58941.png)

_Does moving the call control bar in this Stack avoid initial focus before the first tab?_
Yes, moving the controlBar below the gallery would prevent it from having first initial focus and first tab focus as well. 
Hence why it was moved above the gallery for Call Composite

Reply:
I see. Thanks.

Reply:
to miguel's point here, should `horizontal` prop be removed from this stack then?